docs/source/features/types.rst

```diff
@@ -346,4 +353,6 @@ purposes they all derive from a single type, String. In the middle, in red,
 are the types native to CMakePPLang. Native CMakePPLang types derive 
 from String as well, with "Class" also deriving from "Type". User-defined 
 classes are symbolized by the green box at the bottom, all of which derive 
-from "Object", and may have relationships among themselves as well.
+from "Object", and may have relationships among themselves as well. Not shown
+are pointers. Each type has an associated pointer type, including pointer
```

Comment:
I think a link to the `.. _features-types-other-pointer:` should be added here since they are not shown explicitly here. I tried in this suggestion, but I'm not sure if I got the syntax 100% correct.
```suggestion
are :ref:`pointers <features-types-other-pointer>`. Each type has an associated pointer type, including pointer
```

Reply:
Looks right to me and renders with a proper link